app/views/documents/_publication_attachments.html.erb

```diff
@@ -0,0 +1,19 @@
+<%
+  attachments = document.attachments.for_current_locale
+  title = t('document.headings.attachments', count: attachments.size) unless local_assigns.include?(:title)
+  published_on = '' unless local_assigns.include?(:published_on)
+%>
+<div class="publications-attachments">
```

Comment:
I think the `<section>` that was here before actually made some sense. It's a section with a clear title that can use `labelledby` and it is a "thematic grouping of content". The `<article>` below it confuses me and seems superfluous. If anything it should be an article that wraps the list of attachments rather than a `div` or `section`.

Assuming that the list of documents would make sense on its own, I think the ideal semantics are:
- article
  - h1
  - attachments